proto/v1beta1/vulnerability.proto

```diff
@@ -146,7 +146,14 @@ message Vulnerability {
   // security tracker.
   google.protobuf.Timestamp source_update_time = 6;
 
-  // Next free ID is 7.
+  // This field tracks the CVE record for this vulnerability
+  string cve = 7;
+  // The CWE includes both software & hardware weakness types for this vulnerability.
+  repeated string cwe = 8;
```

Comment:
Are you expecting the CWEs to be treated same as CVEs? Would it make sense to attach vulnerability Occurrences to CWE notes, same as what's done for CVEs?

Reply:
Yes I think having multiple notes for each CWE & then a corr. vulnerability occurence might work. But would that mean we would have multiple vuln occurences with the same `CVSSv3` & maybe `CVSSv2`.

One other concern, if the same resource url has CVE & multiple CWEs do you see any problem when creating occurence or that should be acceptable.

Reply:
IIUC the model you're describing, you'd want `VulnerabilityNote` per each CVE and CWE. Then for a given affected image, you'd create `VulnerabilityOccurrence` with the same `resource_uri` and attached to the relevant `VulnerabilityNote`s that correspond to the CVEs/CWEs. The score is per `VulnerabilityOccurrence`, so yes, they may have similar CVSS v2 and v3 scores across multiple.